File: 1048.c
// adriano r. de sousa

#include <stdio.h>

int main(void)
{

    float salario = 0, reajuste = 0, percentual = 0;

    scanf("%f", &salario);

    if (salario > 0 && salario <= 400)
        percentual = 15;
    else if (salario > 400.01 && salario <= 800)
        percentual = 12;
    else if (salario > 800.01 && salario <= 1200)
        percentual = 10;
    else if (salario > 1200.01 && salario <= 2000)
        percentual = 7;
    else
        percentual = 4;

    reajuste = salario * percentual/100;
    salario += reajuste;
    printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %.0f %%\n", salario, reajuste, percentual);
    return 0;
}

File: 1048.py
#!pyrhon3.8

salario = float(input())
percent = 0

if salario > 0 and salario <= 400:
    percent = 15/100
elif salario > 400 and salario <= 800:
    percent = 12/100
elif salario > 800 and salario <= 1200:
    percent = 10/100
elif salario > 1200 and salario <= 2000:
    percent = 7/100
elif salario > 2000:
    percent = 4/100

#python3.8
# print(f'''Novo salario: {salario+salario*percent:.2f}
# Reajuste ganho: {salario*percent:.2f}
# Em percentual: {percent*100:.0f} %''')

#python3.4
print('Novo salario: {:.2f}\nReajuste ganho: {:.2f}\nEm percentual: {:.0f} %' .format(salario+salario*percent, salario*percent, percent*100))
